generators/server/templates/src/main/resources/_logback-spring.xml

```diff
@@ -61,7 +61,7 @@
         <resetJUL>true</resetJUL>
     </contextListener>
 
-    <root level="@logback.loglevel@">
+    <root level="#logback.loglevel#">
```

Comment:
we need to confirm, the log dynamic setting from the GUI is not affected by any of this. Technically it shouldn't be affected, but im not sure what kind of voodo spring does here